examples/animatediff/train.py

```diff
@@ -294,6 +294,7 @@ def main(args):
 
     if args.dataset_sink_mode and args.sink_size != -1:
         steps_per_sink = args.sink_size
+        total_train_steps = args.epochs * steps_per_sink
```

Comment:
From user point, args.epochs correspond to number of iterations on overall dataset, NOT sink size.

Please adjust the training setting according to this def.

Reply:
fixed